### ISSUE TITLE
 [XamarinProfiler-MAC]: Accessibility: Voiceover: MAS4.2.1: The name property of the "Filter" button is not being announced by the VoiceOver.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#438]
### Steps to Reproduce

1. Follow the pre-requisite to launch the xamarin profiler application.
2. Select the Available instruments to "Allocations/Time Profiler" and invoke next button. Invoke Start Profiling button.
3.This will invoke "Allocations/Time Profiler" Profiling page.
4. Navigate to the Call Tree/Samples tab items and open it.
4. Navigate to the Filter button (i.e., RHS in the profiler window) as shown in the attachment.
5. Observe the VoiceOver behavior for this button.

### Expected Behavior
The VoiceOver should announce as "Filter" button.
### Actual Behavior
The VoiceOver  announces as  button, but Name of the button is not being announced by the VoiceOver.
### User Impact
The User won't be able to know the purpose of the button until its name is not being announced
### Screenshot
![voiceover_profilermac3](https://user-images.githubusercontent.com/22956130/41714713-e5f1fd0c-756e-11e8-9bf4-c83118caca8b.png)

### Environment
macOS High Sierra OSX version: 10.13.4
Visual Studio for Mac Enterprise version: 7.5.2(build 40)
Xamarin Profiler version: 1.6.3-400

**Comment by WiproPSAccTest** [comment]
Please add the following tags to this bug:
A11yMAS
Accessibility
MAS4.2.1
Xamarin-Profiler-MAC
Wipro
